src/themes/icons/generate.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Sorry, but I will leave it to someone else.
As I intend to use that script for my next PR (facebook video) and other changes specific to my system that won't be pushed, I propose to change the order of the icons in the sprite image to be in alphanumeric order and not sporadic as it is now and change the url.png to link.png to have all the icons match the command names.
I can push those changes separately.

Since there is no tool at all to create the sprite image and it have to be done manually, I don't see a reason why not to use it until there will be a better solution proposed by someone else.
I don't intend to create the image manually for my next changes. Any merge in the image will not be possible without a tool.

Reply:
Please activate to allow changes from moderators 
![image](https://user-images.githubusercontent.com/639467/38780943-f75bad9a-40d5-11e8-9b4f-1b02a214eb73.png)
^

Reply:
it is allowed on my side...

Reply:
Thanks.